spec/exchanges/bittrex/integration/market_spec.rb

```diff
@@ -40,4 +40,36 @@
     expect(DateTime.strptime(ticker.timestamp.to_s, '%s').year).to eq Date.today.year
     expect(ticker.payload).to_not be nil
   end
+
+  it 'fetch order book' do
+    btc_eth_pair = Cryptoexchange::Models::MarketPair.new(base: 'BTC', target: 'ETH', market: 'bittrex')
+    order_book = client.order_book(btc_eth_pair)
+
+    expect(order_book.base).to eq 'BTC'
+    expect(order_book.target).to eq 'ETH'
+    expect(order_book.market).to eq 'bittrex'
+    expect(order_book.asks).to_not be_empty
+    expect(order_book.bids).to_not be_empty
+    expect(order_book.asks.first.size).to eq 2
+    expect(order_book.bids.first.size).to eq 2
+    expect(order_book.timestamp).to be_a Numeric
+    expect(order_book.payload).to_not be nil
+  end
+
+  it 'fetch trade' do
+    btc_eth_pair = Cryptoexchange::Models::MarketPair.new(base: 'BTC', target: 'ETH', market: 'bittrex')
+    trades = client.trades(btc_eth_pair)
+    trade = trades.sample
+
+    expect(trades).to_not be_empty
+    expect(trade.trade_id).to_not be_nil
+    expect(trade.base).to eq 'BTC'
+    expect(trade.target).to eq 'ETH'
+    expect(['BUY', 'SELL']).to include trade.type
```

Comment:
@dsantosmerino Saw in your other PR that trade.type will be `buy` or `sell`, can we always make them lowercase?

Reply:
Sure! I was just parsing in raw from the API, but I think that could be better to normalize them.